### DISCUSSION TITLE
Gen AI Course

### DISCUSSION THREAD

**Discussion by MichaelDecent** [swarmauri/swarmauri-sdk#744] [Q&A] [answered]
Q&A discussion forum for the Swarmauri Generative AI Course

**Comment by irshadhuzaifa** [comment]
Where to get the API key for DeepInfra? Required for Module 5.

**Reply by MichaelDecent** [reply, marked as answer]
Create an account via https://deepinfra.com/login?from=%2Fdash then generate your API_KEY
